index.html

```diff
@@ -1344,14 +1334,19 @@ <h3 id="encryption">Encryption</h3>
           >.
         </p>
 
-        <h3 id="multiple_measurements">Multiple measurements of the same type</h3>
+        <h3 id="multiple_measurements">Multiple measurements/events of the same type</h3>
         <p>
           If you want to send multiple measurements of the same type, e.g. three temperatures,
           you can just add multiple measurements of the same type to the payload. A
           postfix will be added to the measurement name (e.g. temperature_2) in the order
           of which you define the measurements. Note that this implies that you will need to use
           the same order in each advertisement, to prevent measurements being assigned to the wrong
           entity. If only one measurement of a certain type is sent, no postfix will be used.
+          <br /><br />
+          For events, a special event `0x00` (None) is available, e.g. in case you have a
```

Comment:
This is now twice in this doc. I think that we can just make this section about just Multiple measurements, and have a multiple events next to the events table.

Reply:
Yeah, I was struggling a bit with this, as the Multiple measurement section was at the end of the page, so after the event section, where I wanted to explain this. I now moved the "multiple measurement" explanation to below the table with all the sensor types, and moved the part about events to below the event types table.